webpack/ForemanRhCloudPages.js

```diff
@@ -4,11 +4,13 @@ import { registerRoutes as foremanRegisterRoutes } from 'foremanReact/routes/Rou
 import ForemanInventoryUpload from './ForemanInventoryUpload';
 import InsightsCloudSync from './InsightsCloudSync';
 import InsightsHostDetailsTab from './InsightsHostDetailsTab';
+import CVEsHostDetailsTab from './CVEsHostDetailsTab';
 
 const pages = [
   { name: 'ForemanInventoryUpload', type: ForemanInventoryUpload },
   { name: 'InsightsCloudSync', type: InsightsCloudSync },
   { name: 'InsightsHostDetailsTab', type: InsightsHostDetailsTab },
+  { name: 'CVEsHostDetailsTab', type: CVEsHostDetailsTab },
```

Comment:
I think this is needed only if you intend to use it from erb. Since we're using it from slot and fill, I think it's not needed here.

@MariaAga , please keep me honest here

Reply:
Yup, componentRegistry is only for erb, right. The component is registered in this pr in `addGlobalFill`